src/components/NavigationHeader.tsx

```diff
@@ -5,16 +5,19 @@ import { ArrowLeftMedium, CloseMedium } from "@ledgerhq/native-ui/assets/icons";
 import { Flex, Text, Link } from "@ledgerhq/native-ui";
 import { StackHeaderProps } from "@react-navigation/stack";
 import { getHeaderTitle } from "@react-navigation/elements";
+import type { FlexBoxProps } from "@ledgerhq/native-ui/components/layout/Flex";
```

Comment:
I know that `import type` is perfectly correct here but can you try removing `type` since it kills the linter for this entire file 💀 ?

Reply:
Done 🏁